services/core-web/src/components/mine/ProjectSummaries/MineProjectSummaryTable.js

```diff
@@ -15,40 +18,70 @@ const propTypes = {
   isLoaded: PropTypes.bool.isRequired,
 };
 
-const transformRowData = (projectSummaries) =>
-  projectSummaries.map((projectSummary) => ({
-    key: projectSummary.project_summary_guid,
-    projectSummary,
-    mine_guid: projectSummary.mine_guid,
-    status_code: projectSummary.status_code,
-    documents: projectSummary.documents,
-    project_summary_id: projectSummary.project_summary_id || Strings.EMPTY_FIELD,
-    update_timestamp: formatDate(projectSummary.update_timestamp),
-  }));
+const transformRowData = (projectSummaries) => {
+  return projectSummaries.map((projectSummary) => {
+    const contact = projectSummary?.contacts?.find((c) => c.is_primary);
+
+    return {
+      key: projectSummary.project_summary_guid,
+      projectSummary,
+      mine_guid: projectSummary.mine_guid,
+      project_stage: projectSummary.status_code,
+      documents: projectSummary.documents,
+      project_summary_id: projectSummary.project_summary_id || Strings.EMPTY_FIELD,
+      project_name: projectSummary.project_summary_title,
+      project_proponent_id: projectSummary.proponent_project_id || Strings.EMPTY_FIELD,
+      project_contact: contact?.name || Strings.EMPTY_FIELD,
+      first_submitted_date: formatDate(projectSummary.create_timestamp),
+      last_updated_date: formatDate(projectSummary.update_timestamp),
+    };
+  });
+};
 
 export const MineProjectSummaryTable = (props) => {
   const columns = [
     {
-      title: "Project ID",
-      dataIndex: "project_summary_id",
-      sorter: true,
-      render: (text) => <div title="Project ID">{text}</div>,
+      key: "project_name",
+      title: "Project name",
+      dataIndex: "project_name",
+      render: (text) => <div title="Project name">{text}</div>,
+    },
+    {
+      key: "project_proponent_id",
+      title: "Project Proponent ID",
+      dataIndex: "project_proponent_id",
+      render: (text) => <div title="Project Proponent ID">{text}</div>,
+    },
+    {
+      key: "project_contact",
+      title: "Project contact",
+      dataIndex: "project_contact",
+      render: (text) => <div title="Project contact">{text}</div>,
     },
     {
+      key: "project_stage",
       title: "Project stage",
-      dataIndex: "status_code",
-      sorter: true,
+      dataIndex: "project_stage",
       render: (text) => (
         <div title="Project stage">
           {props.projectSummaryStatusCodesHash[text] || Strings.EMPTY_FIELD}
         </div>
       ),
     },
     {
-      title: "Last updated",
-      dataIndex: "update_timestamp",
-      sorter: true,
-      render: (text) => <div title="Last updated">{text}</div>,
+      key: "first_submitted_date",
+      title: "First submitted date",
+      dataIndex: "first_submitted_date",
+      render: (text) => <div title="First submitted date">{text}</div>,
+      sorter: dateSorter("first_submitted_date"),
+    },
```

Comment:
with the addition of sorters, what's the default?

Reply:
Updated date is the default as it is the most fresh, if that doesn't seem appropriate please let me know